lib/ansiblelint/rules/__init__.py

```diff
@@ -120,3 +120,5 @@ def matchyaml(self, file, text):
                 matches.append(Match(linenumber,
                                      section, file['path'], self, message))
         return matches
+
+from ansiblelint import RulesCollection  # noqa F401: new class location
```

Comment:
Let's not do imports at the bottom.

Reply:
@webknjaz This is why I asked you to test yourself, that was needed in order to avoid the cyclic-import. This code has a cyclic import but it avoids a runtime failure due to this late-import.